### DISCUSSION TITLE
Editor: Restart from when you last edited your beatmap

### DISCUSSION THREAD

**Discussion by animrei** [ppy/osu#23677] [Ideas] [answered]
I find it a bit inconvenient that as a mapper (a mapper for mania to be specific), I have to navigate through multiple timing points all the time just to reach the part I was previously mapping. 

A feature to start back from when you last edited could help quite a lot, and it could be made by just simply adding a button/keyboard shortcut or so.

Maybe via the right click menu or in the editor there can be a button that shows "Edit from last point" and going back would be pretty simple by (maybe) pressing shift and going rewind? I'm not very sure.

I also noticed a difficulty in implementing this which would be the fact that the game indeed CAN freeze since it's loading too many objects at once.

**Comment by peppy** [comment, marked as answer]
You can use "V" to seek to the last object (and "Z" for the first). But the editor should also be saving the last edit time.

**Reply by peppy** [reply]
Also https://github.com/ppy/osu/pull/23711.
